helper/definitions.py

```diff
@@ -99,3 +99,7 @@ class VideoLinks(str, Enum):
     BACK_TO_NICEDAY = "back to niceday"
     INTRO_EXACUTION_VIDEO = "Intro execution video"
     TESTVIDEOLINK = "testLink"
+
+"""Dictionary for the expected time interval of the dialog"""
+DialogExpectedDuration = {"EXTERNAL_trigger_execution_introduction": "1 6",
+                          "EXTERNAL_trigger_general_activity": "4 9"}

```

Comment:
Maybe it's better to make use of the definition we already have, so to make the code easier to maintain: 

```suggestion
DialogExpectedDuration = { PreparationInterventionComponents.PROFILE_CREATION: "1 6",
                         PreparationInterventionComponents.MEDICATION_TALK ": "4 9"}

```